openweathermap-ios/openweathermap-ios.xcodeproj/project.pbxproj

```diff
@@ -80,7 +84,26 @@
 			children = (
 				5587458F24202F27001A2DE3 /* MapViewController.swift */,
 			);
-			path = ViewControllers;
+			name = ViewControllers;
+			path = ../ViewControllers;
+			sourceTree = "<group>";
+		};
+		55E0C6FF242BDC48004AE9D4 /* Maps */ = {
+			isa = PBXGroup;
+			children = (
+				55E0C700242BDC81004AE9D4 /* Model */,
+				36EC2D57241FB21B00A168F4 /* ViewControllers */,
```

Comment:
It's hard to read those kinds of changes in pbxproj. Consider pasting a screenshot on PR's body in order to present your proposal.

Reply:
Folders were created to organize the classes. It may be that you act equitable.

![image](https://user-images.githubusercontent.com/21261476/77653104-18a7be00-6f4e-11ea-90e1-282c0c6aee7a.png)